packages/elements/src/Header/Header.tsx

```diff
@@ -268,6 +268,7 @@ export function Header(props: Props) {
                       (rightButton ? 72 : 16) +
                       insets.left -
                       insets.right),
+              marginLeft: headerTitleAlign === 'center' ? 12 : 4,
```

Comment:
Why the different left margin for the centered title? It makes it slightly off-center.